### ISSUE TITLE
Allow collections as attributes and operation parameters

### ISSUE THREAD

**Issue by devaigergely81** [ELTE-Soft/txtUML#197]
The plan is to indicate multiplicities of attributes and operation parameters using collections. The discussion on the different collection types is in a separate issue: #59. Currently we only have the `Collection` collection, but we could already try to allow it as attribute and parameter type.
I see three TODOs related to this:
- Change the corresponding validation rule.
- Check if the runtime works correctly. Do you expect any problem, @kovacsgabor?
- Change the UML export to handle this by using `*` multiplicity in the generated model.

**Comment by kovacsgabor** [comment]
> Check if the runtime works correctly. Do you expect any problem, kovacsgabor?

No, I do not. The API already has a ```Collection.Empty``` class which can be instantiated by the user. It is an empty collection which can be extended with its appropriate methods. (Note, however, that as all txtUML collections, ```Collection.Empty``` is immutable and therefore may only be used to create new collection instances.)

**Comment by kovacsgabor** [comment]
After #542, this issue remains a purely validation problem.
